app/build.gradle

```diff
@@ -187,6 +184,34 @@ dependencies {
     androidTestCompile 'com.android.support:support-annotations:25.1.0'
 }
 
+// SHA and timestamp caching courtesy of https://github.com/gdg-x/frisbee/blob/develop/app/build.gradle#L193-L218
+def commitSha() {
```

Comment:
let's leave these here for now -- it's nice to just be able to look into one build.gradle file to see everything that's going on